### ISSUE TITLE
[FR] Wrap text in post window

### ISSUE THREAD

**Issue by madskjeldgaard** [davidgranstrom/scnvim#29]
Text in the post window goes beyond the window, making it a bit hard to read, without resizing. This is a common problem in the SCIDE as well. It would be nice if there was a way to make this text wrap automatically according to screen size. Maybe this is already possible via som nvim magic that I didn't notice? I dunno, otherwise it's an idea for a feature!

**Comment by davidgranstrom** [comment]
Yes, this is possible. The post window actually have its own `filetype` called `scnvim`. So you could make this happen by:

1) Creating an autocmd in you vimrc/init.vim `autocmd FileType scnvim setlocal wrap`
2) Create a file in your runtimepath (usually `.config/nvim`) `nvim/after/ftplugin/scnvim.vim` and add your settings in that file e.g. `setlocal wrap`

**Comment by davidgranstrom** [comment]
This is also something that we should add to the scnvim (vim) documentation. Maybe we could have a "FAQ" section in the docs?

**Comment by madskjeldgaard** [comment]
Sounds perfect. I'll try your idea out – it looks like the stuff!

**Comment by madskjeldgaard** [comment]
Okay I tried 1 and it worked perfectly!

**Comment by kflak** [comment]
Reopening this, as I can't seem to make it work any longer. At some point in the not too distant past, none of the two proposed solutions take any effect. Could there be some runtime voodoo that has changed at some point in the last few months? I have tried both of the above strategies, and none of them work. The output of 
```:verbose setlocal wrap?``` 
is 
```nowrap
        Last set from ~/.local/share/nvim/site/pack/packer/start/scnvim/ftplugin/scnvim.vim line 5
```

**Comment by kflak** [comment]
OK, it seems this might be connected with `packer` and how this loads stuff. If I follow the lead of @madskjeldgaard and do this: 
```
        use {
            'davidgranstrom/scnvim',
            config = function()
               vim.cmd[[autocmd filetype scnvim setlocal wrap]]
            end,
        }
```
things work as they should.

**Comment by kflak** [comment]
This approach seems no longer to work. Can't quite pinpoint when it stopped working...

**Comment by madskjeldgaard** [comment]
This seems to be an issue again

**Comment by kflak** [comment]
Works beautifully! Thanks for the (albeit temporary) fix.

**Comment by elgiano** [comment]
I ran into this as well. My workaround is to add some configurable options to postwin:

<details>

```diff
diff --git a/lua/scnvim/postwin.lua b/lua/scnvim/postwin.lua
index f618b69..f878826 100644
--- a/lua/scnvim/postwin.lua
+++ b/lua/scnvim/postwin.lua
@@ -30,6 +30,9 @@ M.on_open = action.new(function()
   for _, s in ipairs(decorations) do
     vim.opt_local[s] = false
   end
+  for opt, v in pairs(config.postwin.opts) do
+    vim.opt_local[opt] = v
+  end
   vim.opt_local.colorcolumn = ''
   vim.opt_local.foldcolumn = '0'
   vim.opt_local.winfixwidth = true
```

</details>

Then, I can set wrap in scnvim config:
```lua
scnvim.setup {
  postwin = { opts = { wrap = true }  }
}
```

I can make a pr if there is interest in this approach
